lib/matplotlib/image.py

```diff
@@ -1170,84 +997,17 @@ def contains(self, mouseevent):
 
         return inside, {}
 
-    def get_size(self):
-        """Get the numrows, numcols of the input image"""
-        if self._A is None:
-            raise RuntimeError('You must first set the image array')
+    def make_image(self, renderer, magnification=1.0, unsampled=False):
+        bbox_in = self.get_window_extent(renderer).frozen()
+        bbox_in._points /= [renderer.width,
+                            renderer.height]
+        bbox_out = self.get_window_extent(renderer)
+        clip = Bbox([[0, 0], [renderer.width, renderer.height]])
+        self._transform = BboxTransform(Bbox([[0, 0], [1, 1]]), clip)
```

Comment:
`BboxImage` is dicey on all kind of levels...  In practice, `make_image` is only ever called from `draw`, however.